frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -285,14 +285,12 @@ object Project {
     // Project is unique and derived from project name and ivy project configuration
     val hydraTag = project.name
 
-    val hydraRootBaseDir = workspaceDir.resolve("hydra").createDirectories
+    val hydraRootBaseDir = workspaceDir.resolve(".hydra").createDirectories
     val hydraBaseDir = hydraRootBaseDir.resolve("bloop").createDirectories
```

Comment:
@jvican Ideally, we should also set the location of the `hydra.log`, as currently the hydra.log is stored inside the bloop installation folder. The difficulty is that to configure it we use a system property `-Dhydra.logFile`, and since Bloop spawns a single JVM I'm not sure how we can properly set the location as it can be different depending on the project being compiled. In essence, for a given `Project`, here is the value to set for the system property:

```
System.setProperty("hydra.logFile", hydraBaseDir + File.separator + "hydra.log")
```

Do you have any suggestion on how to do it? This is definitely not a blocker, but it'd be nice we sort it out eventually :)

Reply:
Adding a setting is simple, the difficulty is propagating the value so that it's used by logback. I think this is a non trivial effort for little gain, so I'd say we can just keep things as they are and we will document where the hydra log is stored when using Bloop. Let's keep it simple :)

Reply:
Agreed! Let's make sure that the logback setup you have in Hydra is friendly to concurrent Hydra compilations in another build. For example, if I was compiling two projects in two different builds at the same time, would logback correctly redirect the logs of each compilation to independent files?

Reply:
No, right now it would all end up in the same log file. But I think we can live with this limitation. I believe the alternative would be to take advantage of classloaders' isolation and have a different logback.xml in each compilation process, configured so that the hydra.log is stored in the `<project-root>/.hydra/bloop` dir. It's a bit of work and I'd prefer not to make this a blocker. The hydra.log is only relevant if there are issues, and if that occurs we can always guide users to remove any artifact due to concurrent compilation (if that is needed, which I really think it won't be needed in most if not all cases). Sounds good?

Reply:
Another way to work around this is not to use globally configured logging frameworks such as logback. I don't think it's a blocker, but it's definitely nicer to have no global state in the compiler class loader. Specifically, I'm concerned about that global state leaking out to the bloop classloader. It doesn't look like that should happen... so I'm fine proceeding as you point out. But this interaction is something to have in mind in case people start opening tickets about lost messages or logging-related issues.